doc/classes/ProjectSettings.xml

```diff
@@ -937,6 +937,12 @@
 		<member name="input/ui_text_caret_word_right.macos" type="Dictionary" setter="" getter="">
 			macOS specific override for the shortcut to move the text cursor forward one word.
 		</member>
+		<member name="input/ui_text_clear_selection" type="Dictionary" setter="" getter="">
+			If there's only one active caret active with a selection, clears the selection.
+			In case there's more than one caret active, with multiple selections, the selections and secondary carets are first cleared with [code]ui_text_remove_secondary_carets[/code].
+			It's recommend that this shortcut is set the same as the shortcut assigned to [code]ui_text_remove_secondary_carets[/code].
```

Comment:
```suggestion
			In case there's more than one caret active, with multiple selections, the selections and secondary carets are first cleared with [member input/ui_text_remove_secondary_carets].
			It's recommend that this shortcut is set the same as the shortcut assigned to [member input/ui_text_remove_secondary_carets].
```

Reply:
Done, I also fixed a typo "it's recommend".